agenda-api/src/main/java/org/exoplatform/agenda/rest/model/EventSearchResultEntity.java

```diff
@@ -0,0 +1,35 @@
+package org.exoplatform.agenda.rest.model;
+
+import lombok.*;
+import org.exoplatform.agenda.model.EventSearchResult;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EventSearchResultEntity implements Serializable {
+
+  private static final long           serialVersionUID = 6767002872252834666L;
+
+  private String         id;
+
+  private List<String>   excerpts;
+
+  private CalendarEntity calendar;
```

Comment:
Why retrieving all the calendarEntity, which information is required in Search UI ?